ktor-client/ktor-client-core/api/ktor-client-core.klib.api

```diff
@@ -467,10 +468,18 @@ final class io.ktor.client.plugins.sse/DefaultClientSSESession : io.ktor.client.
 }
 
 final class io.ktor.client.plugins.sse/SSEClientContent : io.ktor.http.content/OutgoingContent.ContentWrapper { // io.ktor.client.plugins.sse/SSEClientContent|null[0]
-    constructor <init>(kotlin.time/Duration, kotlin/Boolean, kotlin/Boolean, io.ktor.http.content/OutgoingContent) // io.ktor.client.plugins.sse/SSEClientContent.<init>|<init>(kotlin.time.Duration;kotlin.Boolean;kotlin.Boolean;io.ktor.http.content.OutgoingContent){}[0]
```

Comment:
Should we create deprecated hidden contructor for backward compatibility?

Reply:
`SSEClientContent` is marked with a `@InternalAPI` annotation, so I thought it's not necessary